### PR TITLE
Improvement in not-found page#1698

### PR DESCRIPTION
**Pull request by AshishkrGoyal**
## Description

1. `Back To Page` button is added and Design enhanced.
2. Library `particle.js` has been removed to improve performance.

## Related issues and discussion
#1698 

## Screenshots, if any
image

i18n support - 
image

image

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by bharathcgowda** [comment]
@AshishkrGoyal  can you please check the conflicts?
@josehernandezfintecheandomx  could you please review this PR

**Comment by AshishkrGoyal** [comment]
@bharathcgowda for me it is not showing conflicts with the base branch i.e. master .
can you please guide in which file you can see conflicts ?

**Comment by AshishkrGoyal** [comment]
> Can you follow the [commit message conventions](https://github.com/openMF/web-app/blob/master/.github/COMMIT_MESSAGE.md)?

@PC-11-00 thanks for it, I have checked it and will follow it in next commits . 
If it is an issue then i can raise a new PR, that's ok for me.

File: src/app/not-found/not-found.component.html [hunk 1]
```diff
@@ -1,5 +1,8 @@
-  <div id="particles-js"></div>
-  <div class="text">
-    <h1 id="title">404</h1>
-    <p id="subtitle">Something went wrong.</p>
+  <div class="not-found-container">
+    <div id="title">404</div>
+    <div id="subtitle" class="f30">Oops.. Looks like this page doesn't exist.</div>
```
[the remark this answers is not in the document]
Comment (AshishkrGoyal) [reply src/app/not-found/not-found.component.html]: @josehernandezfintecheandomx this is fixed, thanks for the nice suggestion.

File: src/app/not-found/not-found.component.html [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+    <div class="back-button m-t-20">
+      <button type="button" (click)="goBack()" mat-raised-button>
+        <fa-icon icon="arrow-left" class="m-r-10"></fa-icon>Back To Page</button>
```
[the remark this answers is not in the document]
Comment (AshishkrGoyal) [reply src/app/not-found/not-found.component.html]: @josehernandezfintecheandomx this is fixed.

File: src/app/not-found/not-found.component.scss [hunk 3]
```diff
@@ -1,28 +1,16 @@
...
-	transform: translate(50%,-50%);
-	max-width: 90%;
-	padding: 2em 3em;
-  color: white;
-	font-family: 'Raleway', sans-serif;
   display: flex;
   flex-direction: column;
+  justify-content: center;
   align-items: center;
+  background: cornflowerblue;
```
[the remark this answers is not in the document]
Comment (AshishkrGoyal) [reply src/app/not-found/not-found.component.scss]: @josehernandezfintecheandomx now, we are using colors that we have already defined in color scheme.

**Review by josehernandezfintecheandomx — APPROVED** [review]
LGTM

**Pull request merged**